lua/core/controlspec.lua

```diff
@@ -181,11 +181,11 @@ ControlSpec.PHASE = ControlSpec.new(0, math.pi, 'lin', 0, 0, "")
 --- converts to an exponential RQ range
 ControlSpec.RQ = ControlSpec.new(0.001, 2, 'exp', 0, 0.707, "")
 --- converts to a MIDI range (default 64)
-ControlSpec.MIDI = ControlSpec.new(0, 127, 'lin', 0, 64, "")
+ControlSpec.MIDI = ControlSpec.new(0, 127, 'lin', 1, 64, "")
```

Comment:
suggested:
`ControlSpec.MIDI = ControlSpec.new(0, 127, 'lin', 1, 64, "", 1/127)`

Reply:
Since it's between 0 and 127, shouldn't it be 1/128?

Reply:
it's a good q, but no, quantum is `1/max-min`.

eg. running this code:
```lua
function init()
  params:add{
        type = "control",
        id = "root_note",
        name = "root note",
        controlspec = controlspec.new(0, 11, 'lin', 1, 0, "", 1/12), -- 1/12 vs 1/11
    }
  params:set_action('root_note', function(x) print(x) end )
end
```

repeated `params:delta('root_note',1)` print:
```
1
2
3
4
5
5
```

whereas following `1/max-min` and using 1/11 for the quantum gets us the expected increments.